packages/imperative/src/utilities/src/TextUtils.ts

```diff
@@ -274,7 +274,7 @@ export class TextUtils {
      * @returns {string} - a formatted string with the variables inserted
      */
     public static formatMessage(message: string, ...values: any[]): string {
-        if (!isNullOrUndefined(values)) {
+        if (!(values == null)) {
```

Comment:
There are multiple occasions (in this file and maybe others) where we do `!(a == null)` instead of `a != null` Curious if we could replace them with the shorthand version of `!= null` 😋